src/orquestra/quantum/measurements/measurements.py

```diff
@@ -0,0 +1,356 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
+################################################################################
+from __future__ import annotations
+
+import copy
+import json
+from collections import Counter
+from typing import Dict, Iterable, List, Optional, Sequence, TextIO, Tuple, cast
+
+import numpy as np
+
+from orquestra.quantum.distributions import MeasurementOutcomeDistribution
+from orquestra.quantum.typing import AnyPath
+from orquestra.quantum.utils import (
+    convert_tuples_to_bitstrings,
+    sample_from_probability_distribution,
+)
+
+from .expectation_values import ExpectationValues
+from .parities import check_parity_of_vector
+
+
+def convert_bitstring_to_int(bitstring: Sequence[int]) -> int:
+    """Convert a bitstring to an integer.
+
+    Args:
+        bitstring (list): A list of integers.
+    Returns:
+        int: The value of the bitstring, where the first bit in the least
+            significant (little endian).
+    """
+    return int("".join(str(bit) for bit in bitstring[::-1]), 2)
+
+
+def _convert_bitstrings_to_vector(bitstrings: Iterable[str]) -> np.ndarray:
+    """Converts bitstrings to vector so parity can be checked."""
+    n_qubits = len([*bitstrings][0])
+    all_bits = "".join(bitstrings)
+    bitstring_1d_array = np.frombuffer(all_bits.encode("utf-8"), "u1") - ord("0")
+    return bitstring_1d_array.astype(int).reshape(-1, n_qubits)
+
+
+def get_expectation_value_from_frequencies(
+    marked_qubits: Iterable[int], bitstring_frequencies: Dict[str, int]
```

Comment:
That's what I thought initially as well, but I decided to put `get_expectation_value_from_frequencies` in `measurements.py` because 1) it is only used by `get_expectation_values` in the `Measurements` class, 2) it just returns a float, not a `ExpectationValues` object, so I don't think it necessarily needs to be in the `expectation_values.py`, and 3) it uses another function defined in measurements, `_convert_bitstrings_to_vector`.